src/components/organisms/DMListItem/DMListItem.tsx

```diff
@@ -0,0 +1,197 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import { makeStyles } from '@material-ui/core/styles';
+import Typo from '../../atoms/Typo/Typo';
+import { DialogProps } from '../../../utils/hooks/useDialog';
+import ListItem from '../../atoms/ListItem/ListItem';
+import { MessageType } from '../../../types/Chat';
+import Avatar from '../../atoms/Avatar/Avatar';
+import { UserStatusType } from '../../../types/User';
+import Button from '../../atoms/Button/Button';
+
+type StyleProps = { status: UserStatusType };
```

Comment:
```suggestion
type StyleProps = { status: UserStatusType } | null;
```

> 유저의 status를 가져오기 위해 useStyles에 status를 넣었는 데, 이 status값을 skeleton에서 사용하지 못하기에 skeleton의 useStyles를 따로 작성했습니다.

useStyles의 인자가 `{ status: UserStatusType }` type으로 고정되어있기 때문입니다. 하나의 makeStyles를 이용하시려면 위 suggestion처럼 수정하시고 [이 라인](https://github.com/404-DriverNotFound/frontend-b/pull/91/files#diff-32cdbba47ef831f15b435a1c256280962c6d21efa0b89fbb9b91e7392229287eR91)에서 `useStyles()`를 쓰는 방식으로 해결할 수 있을 것 같습니다. 다만 굳이 고칠 필요는 없는 부분이라는 생각이 들어서, 수정하셔도 안 하셔도 괜찮을 것 같아요.

Reply:
오 어떻게 해야 같이 사용할 수 있을 지 많이 고민했었는 데 친절하게 알려주셔서 감사합니다. skeleton은 따로 export해서 skeleton style은 따로 두었습니다. 감사합니다!!